intermediate/indexing/advanced-indexing.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>As a newbie to advanced indexing: it seems to me that in both orthogonal and vectorized indexing a "different name than the original dimensions" needs to be specified. Instead, the critical part for triggering vectorized indexing is that ALL selections need to target the SAME selection dimension. So for orthogonal, we create two dimensions <code>degrees_north</code> and <code>degrees_east</code>, while in vectorized we create a single <code>points</code> dimension.</p><p> </p><p>Assuming my understanding is correct, how about: "To trigger vectorized..., you will need to provide the selection dimensions with a single shared output dimension name that is different than each of the original dimensions. This dimension name...". I fully appreciate how difficult this is to articulate clearly!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
@JessicaS11 You are correct! Thanks so much for your suggestions. My original wordings was confusing. Using your suggestion, I updated this sentence as following:

To trigger vectorized indexing behavior, you will need to provide the selection dimensions with a new **shared** output dimension name. This means that the dimensions of both indexers must be the same, and the output will have the same dimension name as the indexers.

I made sure to emphasize on the *same* dimension name in a few places.